src/main/java/slimeknights/tconstruct/library/modifiers/Modifier.java

```diff
@@ -324,6 +330,41 @@ public void onInventoryTick(IModifierToolStack tool, int level, World world, Liv
   }
 
 
+  /**
+    * Called when this item is used when targeting a Block
+   * @param tool 
+   * @param level 
+   * @param stack 
+   * @param context        Full item use context
+   * @return  Action result
+   */
+  public ActionResultType onItemUse(ToolStack tool, int level, ItemStack stack, ItemUseContext context) {
+    return ActionResultType.PASS;
+  }
+  
+  public ActionResultType itemInteractionForEntity(ToolStack tool, int level, ItemStack stack, PlayerEntity playerIn, LivingEntity target, Hand hand) {
```

Comment:
All public methods should get either javadocs or `@Override`, especially in library. For these specifically, do the alternatives list like I did above, and note somewhere which order they run in. Also note the return value for pass and to prevent interaction